Sofa/Component/Constraint/Lagrangian/Correction/src/sofa/component/constraint/lagrangian/correction/UncoupledConstraintCorrection.inl

```diff
@@ -29,6 +29,30 @@
 #include <sofa/core/topology/TopologyData.inl>
 #include <sofa/core/ConstraintParams.h>
 
+namespace // anonymous
```

Comment:
Move to `namespace sofa::type::trait` in `Sofa.Type`

Reply:
I did not do anything in that part 😅 but I can see the traits `is_container` and `is_vector` are in the `sofa::type::trait` namespace 🫢...
But Rebind is not, indeed, so I guess it should be done in an other PR